README.md

```diff
@@ -27,37 +27,6 @@ where `PATH` is a file or directory of files to be rewritten.
 
 The Get-Originals Rewriter provides the following options:
```

Comment:
This sentence is now kind of dangling. You want to mention the usage as either file or directory.

Reply:
I just removed it, since I address that file / directory usage in the sentence before. The README is in major need of an update generally, I think I'll have a PR for that next week.